src/include/distributed/multi_executor.h

```diff
@@ -26,6 +26,14 @@ typedef enum
 	SEQUENTIAL_CONNECTION = 1
 } MultiShardConnectionTypes;
 
+/* whether to use remote transaction blocks */
+typedef enum TransactionBlocksUsage
```

Comment:
These are specifically remote transactions, so should we have a `REMOTE` prefix maybe?

Reply:
It messed with alignment a lot, so was looking for a way to make it shorter.